src/stylesheets/theme/_sds-theme-general.scss

```diff
@@ -27,4 +27,5 @@ https://v2.designsystem.digital.gov/style-tokens
         padding-left: (units($theme-site-margins-width));
         padding-right: (units($theme-site-margins-width));
     }
-  }
\ No newline at end of file
+  }
+$button-shadow:0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
```

Comment:
Can a USWDS Shadow Token be used instead?

Reply:
no because we are chaining the shadows. The first one produces the border

Reply:
this has been resolved